scripts/xcode_select_current_version.sh

```diff
@@ -3,7 +3,7 @@
 if [ -n "$XCODE_PATH_OVERRIDE" ]; then # If someone calls this with the XCODE_PATH_OVERRIDE variable set to a path to a developer dir, use it instead
     XCODE_PATH="$XCODE_PATH_OVERRIDE"
 else
-    XCODE_PATH='/Applications/Xcode_12.4.app/Contents/Developer'
+    XCODE_PATH='/Applications/Xcode_12.5.1.app/Contents/Developer'
```

Comment:
is 12.5.1 vs 12.5 important?

Reply:
12.5 (12E262) is the default:
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode

I think we should keep that version since it's also the blessed one being used in devmain so we keep consistency where we can.

Reply:
I believe it just the most up to date version that needs minimum OS 11?

Reply:
Yeah I was originally thinking "well might as go with the newest available" but there's definitely value to sticking with devmain, so 12.5 it is!